templates/openstack/install.html

```diff
@@ -335,881 +138,26 @@ <h2>Multi-node OpenStack deployment</h2>
-|           |              |                                                                          |
-| placement | placement    | RegionOne                                                                |
-|           |              |   admin: http://172.16.7.175:8778                                        |
-|           |              | RegionOne                                                                |
-|           |              |   internal: http://172.16.7.175:8778                                     |
-|           |              | RegionOne                                                                |
-|           |              |   public: http://172.16.7.175:8778                                       |
-|           |              |                                                                          |
-| glance    | image        | RegionOne                                                                |
-|           |              |   public: http://172.16.7.167:9292                                       |
-|           |              | RegionOne                                                                |
-|           |              |   internal: http://172.16.7.167:9292                                     |
-|           |              | RegionOne                                                                |
-|           |              |   admin: http://172.16.7.167:9292                                        |
-|           |              |                                                                          |
-+-----------+--------------+--------------------------------------------------------------------------+</code></pre>
-
-        <p>In order to create tenants, networks, launch instances, etc., refer to the python-openstackclient <a href="https://docs.openstack.org/python-openstackclient/latest/cli/command-list.html" class="p-link--external">documentation</a>.</p>
-        </div>
-      </li>
-      </ol>
+      <h2>Large-scale</h2>
+      <p class="u-sv3">NOTE: In order to qualify for the <a href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf" class="p-link--external">Private Cloud Build (PCB)</a> service, <a href="https://assets.ubuntu.com/v1/1a8fb1b3-UA-I_datasheet_2019-Oct.pdf" class="p-link--external">Ubuntu Advantage for Infrastructure (UA-I)</a> support subscription and <a href="https://assets.ubuntu.com/v1/a797f6f9-Datasheet_Openstack_05.10.20+%281%29.pdf" class="p-link--external">Managed OpenStack</a>, at least 12 nodes are required for Charmed OpenStack. Please contact your Canonical sales representative for detailed information on minimum hardware requirements</p>
+      <p class="u-sv3">Always up-to-date copy of the “Installation” section of the <a href="https://docs.openstack.org/project-deploy-guide/charm-deployment-guide/latest/" class="p-link--external">OpenStack Charms Deployment Guide</a> in a PDF form; 1st page screenshot to go here</p>
```

Comment:
There is a small difference from the copydoc here but could be due to the pdf.

Reply:
Not sure which difference you mean

Reply:
![image](https://user-images.githubusercontent.com/58276363/127173805-d36379c0-c60c-4775-a78a-511831e7632b.png)
The text above the call to action here

Reply:
yeah, that should be there thanks.